org.opent2t.sample.hub.superpopular/com.insteon.hub/js/tests/test.js

```diff
@@ -0,0 +1,37 @@
+// var schema = require("c:/githome/translators/org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular");
+// console.log(schema);
```

Comment:
Fixed.

---
In reply to: [82905729](https://github.com/openT2T/translators/pull/107#discussion_r82905729) [](ancestors = 82905729)